src/Adapter/Driver/Pdo/Pdo.php

```diff
@@ -37,6 +37,12 @@ class Pdo implements DriverInterface, DriverFeatureInterface, Profiler\ProfilerA
     /** @var array */
     protected $features = [];
 
+    /**
+     * @var Profiler\ProfilerInterface
```

Comment:
ok, one last bit (sorry)! 

run `vendor/bin/phpcbf` on this and it should fix the CS issue (probably just sprtijg issue)

Reply:
Done